go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/stretchr/testify v1.6.1
 	go.uber.org/zap v1.15.0
 	golang.org/x/crypto v0.0.0-20201016220609-9e8e0b390897 // indirect
+	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a // indirect
```

Comment:
I don't think we need this update. If we need to use `go get` for our tooling, I believe we can adopt the `go.tools.mod` strategy instead. Every tool we need to install locally will use another mod file, leaving the project's file clean.

Reply:
Do you mean to introduce a tools.go file in our repo and add all our tools there? something like [this](https://marcofranssen.nl/manage-go-tools-via-go-modules/)

Reply:
No, I mean this: https://github.com/golang/go/issues/25922#issuecomment-756675225.

In this example you linked, we ensure that the modules are in our mod file. And **don't want that**. So for tooling, we would use this command instead: `go get -modfile=go.tools.mod <the tool>`.

Reply:
Oh okay, so do you want me to update all our scripts in which `go get tool` if not found or just revert the changes of `go.mod` and `go.sum` in this PR and have a followup?
^^ @ricardozanini

Reply:
I think it's ok to add the `go.tools.mod` in this PR. It's a simple task. wdyt?

Reply:
Yeah it'll be okay. I'll add this option everywhere we do `go get `.  not sure if the mod file will be created at this moment from my system since all the tools are already present :)

Reply:
That's a good point. You can always clean up your local `$GOPATH`. hahahah

Reply:
Did that :p